recipes/default.rb

```diff
@@ -246,21 +250,11 @@
   code <<-EOH                                                                                                       
        openssl rsa -in #{node['kagent']['certs_dir']}/priv.key -out #{node['kagent']['certs_dir']}/priv.key.rsa
        chmod 640 #{node['kagent']['certs_dir']}/priv.key.rsa
-       chown root:#{node['kagent'['certs_group']]} #{node['kagent']['certs_dir']}/priv.key.rsa
+       chown #{node['kagent']}:#{node['kagent'['certs_group']]} #{node['kagent']['certs_dir']}/priv.key.rsa
```

Comment:
The user is wrong and in the group there is a missing right bracket

Reply:
thanks